From e068ea54210aaac1b881da9c648469a277ebfd24 Mon Sep 17 00:00:00 2001
From: Russ Weight <russell.h.weight@intel.com>
Date: Wed, 9 Mar 2016 16:23:23 -0800
Subject: [PATCH 1/2] EM/PMIC_CCSM: REVERTME: Restore constant definition

ACINPUTICC1500VAL is no longer used, but it does exist in the git source.
For the short term, add the definition back in, in order to make the
quilt and git source equivalent

Change-Id: I99d4333ad5a38ef181b6e480a524e933e2cb5630
Signed-off-by: Russ Weight <russell.h.weight@intel.com>
---
 drivers/power/intel_pmic_ccsm.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/power/intel_pmic_ccsm.c b/drivers/power/intel_pmic_ccsm.c
index 1cddedc..52453cf 100644
--- a/drivers/power/intel_pmic_ccsm.c
+++ b/drivers/power/intel_pmic_ccsm.c
@@ -52,6 +52,7 @@
 #define USB_WAKE_LOCK_TIMEOUT	(5 * HZ)
 
 #define USBINPUTICC100VAL	100
+#define ACINPUTICC1500VAL	1500
 #define CDP_INPUT_CURRENT_LIMIT 1500
 #define HIGH_POWER_CHRG_CURRENT 2000
 #define LOW_POWER_CHRG_CURRENT 500
-- 
1.9.1

